### PR TITLE
releng: fix wrong image directory

### PR DESCRIPTION
**Pull request by cpanato**
A wrong image directory was introduced in the job created by this PR https://github.com/kubernetes/test-infra/pull/21344

it should use `images/k8s-cloud-builder` instead of `images/releng/k8s-ci-builder`.

The `images/releng/k8s-ci-builder` is the next job to be created.
/kind bug
/assign @justaugustus @saschagrunert @hasheddan

### CODE REVIEW

**Review by hasheddan — APPROVED** [review]
/lgtm

**Comment by cpanato** [comment]
after gets merged I will create a temporary PR to double-check the job

**Review by justaugustus — CHANGES_REQUESTED** [review]
/hold

@cpanato -- Now that we know this pattern works, let's do two things in this PR:

1. Create all jobs for the outstanding presubmits
2. Making the presubmits blocking

**Comment by justaugustus** [comment]
(oooof, too slow I guess)

**Comment by cpanato** [comment]
@justaugustus will do in the next PR

**Pull request merged**
